roles/sft-server/templates/vhost_certbot-http-challenge.conf.j2

```diff
@@ -0,0 +1,17 @@
+server {
```

Comment:
`vhost_certbot-http-challenge.conf.j2`

Odd file naming containing both underscores and dashes. Use underscores or dashes, but not both?

Reply:
I commented on this [further above](https://github.com/wireapp/ansible-sft/pull/1#discussion_r457601987). Please let me know there what you think :)